repository-data/development/src/main/resources/hcm-config/main.yaml

```diff
@@ -2,7 +2,6 @@
 definitions:
   config:
     /hippo:configuration/hippo:modules/autoexport/hippo:moduleconfig:
-      autoexport:enabled: false
```

Comment:
This change will impact the teams established workflow. It's therefore better not to commit this.

Reply:
Good spot - I missed that!

Reply:
Addressed in #45667b4